packages/core/src/App/app.jsx

```diff
@@ -102,6 +133,7 @@ const App = ({ root_store }) => {
                     </DesktopWrapper>
                     <AppModals url_action_param={url_params.get('action')} />
                     <SmartTraderIFrame />
+                    {show_cookie_banner && <CookieBanner onAccept={onAccept} is_open={show_cookie_banner} />}
```

Comment:
Looks like a double check for showing the banner.

Reply:
First one is to decide either to show the banner or not. Second one is for the banner itself to show some animations while opening/closing